<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditorDialog</class>
 <widget class="QDialog" name="EditorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>778</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Editor - ScreenCloud</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QWidget" name="leftToolbar" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>35</width>
         <height>16777215</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QPushButton" name="arrowBtn">
          <property name="toolTip">
           <string>Draw an arrow</string>
          </property>
          <property name="icon">
           <iconset resource="../../res/icons.qrc">
            <normaloff>:/editor/icons/editor/hi16-action-toolbox_arrow.png</normaloff>:/editor/icons/editor/hi16-action-toolbox_arrow.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="boxBtn">
          <property name="toolTip">
           <string>Draw a box</string>
          </property>
          <property name="icon">
           <iconset resource="../../res/icons.qrc">
            <normaloff>:/editor/icons/editor/hi16-action-toolbox_box.png</normaloff>:/editor/icons/editor/hi16-action-toolbox_box.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ellipseBtn">
          <property name="toolTip">
           <string>Draw an ellipse</string>
          </property>
          <property name="icon">
           <iconset resource="../../res/icons.qrc">
            <normaloff>:/editor/icons/editor/hi16-action-toolbox_ellipse.png</normaloff>:/editor/icons/editor/hi16-action-toolbox_ellipse.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="textBtn">
          <property name="toolTip">
           <string>Write a text</string>
          </property>
          <property name="icon">
           <iconset resource="../../res/icons.qrc">
            <normaloff>:/editor/icons/editor/hi16-action-toolbox_text.png</normaloff>:/editor/icons/editor/hi16-action-toolbox_text.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="numberedBtn">
          <property name="toolTip">
           <string>Draw a number in a circle</string>
          </property>
          <property name="icon">
           <iconset resource="../../res/icons.qrc">
            <normaloff>:/editor/icons/editor/hi16-action-toolbox_number.png</normaloff>:/editor/icons/editor/hi16-action-toolbox_number.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="SnapshotCanvas" name="snapshotCanvas">
       <property name="renderHints">
        <set>QPainter::Antialiasing|QPainter::HighQualityAntialiasing|QPainter::TextAntialiasing</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="rightToolbar" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QWidget" name="propertyToolbar" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QColorButton" name="colorBtn">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="toolTip">
              <string>Edit item color</string>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
             <property name="color" stdset="0">
              <color>
               <red>225</red>
               <green>50</green>
               <blue>50</blue>
              </color>
             </property>
             <property name="defaultColor" stdset="0">
              <color>
               <red>225</red>
               <green>50</green>
               <blue>50</blue>
              </color>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="formatTextBtn">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="toolTip">
              <string>Edit item font</string>
             </property>
             <property name="icon">
              <iconset resource="../../res/icons.qrc">
               <normaloff>:/editor/icons/editor/hi16-action-draw_text.png</normaloff>:/editor/icons/editor/hi16-action-draw_text.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="sizeWidget" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QSlider" name="widthSlider">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="toolTip">
                 <string>Edit item size</string>
                </property>
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>9</number>
                </property>
                <property name="value">
                 <number>5</number>
                </property>
                <property name="sliderPosition">
                 <number>5</number>
                </property>
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="tickPosition">
                 <enum>QSlider::TicksBothSides</enum>
                </property>
                <property name="tickInterval">
                 <number>2</number>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SnapshotCanvas</class>
   <extends>QGraphicsView</extends>
   <header>editor/snapshotcanvas.h</header>
  </customwidget>
  <customwidget>
   <class>QColorButton</class>
   <extends>QPushButton</extends>
   <header>editor/qcolorbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../res/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EditorDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EditorDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
